spec/models/rbac_spec.rb

```diff
@@ -1098,7 +1098,7 @@ def get_rbac_results_for_and_expect_objects(klass, expected_objects)
                                                 :miq_group      => group,
                                                 :results_format => :objects)
         expect(results.length).to eq(2)
-        expect(attrs[:total_count]).to eq(3)
+        expect(attrs[:total_count]).to eq(2)
```

Comment:
yay. I really like this returning 2.
since the exact value is only used as a heads up for the user, it doesn't really matter.
But for some reason, returning 2 hear makes me smile.

(0% snark)